### PR TITLE
Azure Application Insights: Add option to ignore null values

### PR DESCRIPTION
**Pull request by yardenshoham**
Added the option to ignore errors during the extraction of Azure Application Insights values

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Relates to https://github.com/kedacore/keda/issues/4316
Relates to https://github.com/kedacore/keda/pull/4329

### CODE REVIEW

File: content/docs/2.10/scalers/azure-app-insights.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ triggers:
     appInsightsResourceURL: https://api.applicationinsights.airgap.io/
     # Required when cloud = Private.
     activeDirectoryEndpoint: https://login.airgap.example/
+    ignoreNullValues: false # Default is `true`, which means ignoring any errors with data extraction from a successful query. Set to `false` the scaler will return error when null values are discovered
```
Comment (JorTurFer) [inline content/docs/2.10/scalers/azure-app-insights.md:33]: This changes the current behavior (now we raise an error), we should maintain current behaviour

File: content/docs/2.10/scalers/azure-app-insights.md [hunk 2]
```diff
@@ -50,6 +51,7 @@ for further details.
 - `cloud` - Name of the cloud environment that the Azure Application Insights instance belongs to. (Values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`, `Private`, Default: `AzurePublicCloud`, Optional)
 - `appInsightsResourceURL` - Application Insights REST API URL of the cloud environment. (Required when `cloud` is set to `Private`, e.g. `https://api.applicationinsights.azure.cn/` for `AzureChinaCloud`).
 - `activeDirectoryEndpoint` - Active Directory endpoint of the cloud environment. (Required when `cloud` is set to `Private`, e.g. `https://login.chinacloudapi.cn/` for `AzureChinaCloud`).
+- `ignoreNullValues` - Ignore any errors with data extraction from a successful query (Values: `true`,`false`, Default: `true`, Optional).
```
Comment (JorTurFer) [inline content/docs/2.10/scalers/azure-app-insights.md:54]: This changes the current behavior (now we raise an error), we should maintain current behaviour

File: content/docs/2.10/scalers/azure-app-insights.md (hunk 1 above)
Comment (yardenshoham) [reply content/docs/2.10/scalers/azure-app-insights.md]: Done

File: content/docs/2.10/scalers/azure-app-insights.md (hunk 2 above)
Comment (yardenshoham) [reply content/docs/2.10/scalers/azure-app-insights.md]: Done

**Review by JorTurFer — APPROVED** [review]

File: .gitpod.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# This configuration file was automatically generated by Gitpod.
```
Comment (tomkerkhove) [inline .gitpod.yml:1]: Can you remove it please? I don't think we need this?

**Review by tomkerkhove — APPROVED** [review]

File: .gitpod.yml (hunk 3 above)
Comment (tomkerkhove) [reply .gitpod.yml]: No worries!

**Review by zroubalik — APPROVED** [review]

**Pull request merged**